Adapters/GraphicsAdapter.cs

```diff
@@ -120,14 +120,15 @@ FormattedText GetText(string str, RFont font)
             return new FormattedText
             {
                 Text = str,
-                Typeface = new Typeface(f.Name, font.Size, f.FontStyle, f.Weight),
+                Typeface = new Typeface(f.Name, /*font.Size, */f.FontStyle, f.Weight),
```

Comment:
Hi!
In 0.10 Avalonia you should use FontManager.Current.GetOrAddTypeface instead of Typeface ctor. It will reuse typefaces.
As an example - https://github.com/AvaloniaUI/AvaloniaEdit/pull/110/files

Reply:
Good to know, thanks. I'll see to that ASAP.